scripts/update-tf-version-in-repository.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# Copyright 2024 MongoDB Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -euo pipefail
+
+# example value of TF_VERSIONS_ARR='["1.9.x", "1.8.x", "1.7.x"]'
+TF_VERSIONS_ARR=$(./scripts/get-terraform-supported-versions.sh "false")
```

Comment:
[nit] would be nice to use `./scripts/get-terraform-supported-versions.sh "latest"` and do the LATEST_TF_VERSION in the script